RecoParticleFlow/PFTracking/plugins/GoodSeedProducer.h

```diff
@@ -147,6 +171,7 @@ class GoodSeedProducer final : public edm::stream::EDProducer<> {
 	
       ///READER FOR TMVA
       std::array<std::unique_ptr<TMVA::Reader>,9> reader{};
+      std::array<std::unique_ptr<GBRForest>,9> gbr;
```

Comment:
Deprecated code needs to be removed.
Since this is a test PR, I'll clean it up later.

-L

(Sent from my Nexus 6)
On Jul 11, 2015 3:02 PM, "Chris Jones" <notifications@github.com> wrote:

> In RecoParticleFlow/PFTracking/plugins/GoodSeedProducer.h
> <https://github.com/cms-sw/cmssw/pull/10143#discussion_r34412459>:
>
> > @@ -147,6 +171,7 @@ class GoodSeedProducer final : public edm::stream::EDProducer<> {
> >  	
> >        ///READER FOR TMVA
> >        std::array<std::unique_ptr<TMVA::Reader>,9> reader{};
> > +      std::array<std::unique_ptr<GBRForest>,9> gbr;
>
> Do you want these 9s to be goodseedhelpers::HeavyObjectCache::kMaxWeights
> or is it just a coincidence?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10143/files#r34412459>.
>